### ISSUE TITLE
Shell parameter expansion not working for /

### ISSUE THREAD

**Issue by camabeh** [gliderlabs/docker-alpine#401]
Hello, I am trying to escape variable which will be used in `sed` command but unfortunately it looks like there is some bug in handling parameter expansion in busybox shell.

Expected:
> Bash in Ubuntu:
```sh
$ var="/Users/Documents/name/file"; echo "${var//\//\\/}"
\/Users\/Documents\/name\/file
```

Actual:
> Shell in Alpine:
```sh
$ var="/Users/Documents/name/file"; echo "${var//\//\\/}"
/Users/Documents/name/file
```

It's wierd because parameter expansion with replacement is working for other characters.

> Bash in Ubuntu:
```sh
$ var="/Users/Documents/name/file"; echo "${var//e/E}"
/UsErs/DocumEnts/namE/filE
```
> Shell in Alpine:
```sh
$ var="/Users/Documents/name/file"; echo "${var//e/E}"
/UsErs/DocumEnts/namE/filE
```

Env:
BusyBox v1.27.2 (2017-11-28 16:17:30 GMT) multi-call binary.
Alpine 3.7

Ref:
https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion

**Comment by ncopa** [comment]
busybox ash is not GNU bash. There are some bash extensions that are implemented but many are not and as you discovered, some does not work same way. You can report it to busybox and ask them to add support for \/ because this seems to be handy to have, but there are no guarantees that they will.

What you can expect though is POSIX compatibility so anything documented in POSIX should work.

Ref: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

**Comment by ncopa** [comment]
If you really need full bash compatibility you can always `apk add bash`

**Comment by camabeh** [comment]
Ok thank you.

**Comment by camabeh** [comment]
It will be fixed in 1.29.x.

https://bugs.busybox.net/show_bug.cgi?id=10901
